### PR TITLE
add trigger to update mat view

### LINKED ISSUE #25: update materialized view when orgs or grants change

### PR DESCRIPTION
**Pull request by bnchdrff**
fixes #25

### CODE REVIEW

File: integration-tests/test-queries/more-meta.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+export const query = `
+query foo {
```
Comment (hampelm) [inline integration-tests/test-queries/more-meta.js]: Tiny nitpick, doesn't block anything, but would be good to have a descriptive name here so anyone can instantly tell what this for

File: src/db/models/organizationMeta.ts [hunk 2]
```diff
@@ -26,6 +30,22 @@ export default (sequelize: Sequelize.Sequelize) => {
       uuid: {
         type: Sequelize.UUID,
       },
+      countGrantsFrom: {
```
Comment (hampelm) [inline src/db/models/organizationMeta.ts]: Another minor question: for the purpose of grouping related fields, would it be clearer to do this as `grantsFromCount`, `grantsToCount` etc (general category first, specific variants later)?

Comment (hampelm) [reply src/db/models/organizationMeta.ts]: I guess this comment belonged in #24

Comment (bnchdrff) [reply src/db/models/organizationMeta.ts]: we can handle this in another pr!

File: integration-tests/test-queries/more-meta.js (hunk 1 above)
Comment (bnchdrff) [reply integration-tests/test-queries/more-meta.js]: the filename says it all imo

**Pull request merged**